tests/unit/test_hnswlib.cpp

```diff
@@ -1189,7 +1188,7 @@ TEST_F(HNSWLibTest, hnsw_serialization) {
     VecSimIndex_DeleteVector(new_index, (size_t)(distrib(rng) * n));
 
     // Persist index, delete it from memory and restore.
-    serializer.saveIndex(file_name);
+    serializer.saveIndex_v1(file_name);
```

Comment:
remove the comment in line 1188, it is not valid anymore, 
is there a scenario that checks block removal + serialization?